src/ansiblelint/config.py

```diff
@@ -9,6 +9,7 @@
     # Do not sort this list, order matters.
     {"jinja2": "**/*.j2"},  # jinja2 templates are not always parsable as something else
     {"jinja2": "**/*.j2.*"},
+    {"text": "**/templates/**/*.*"},  # templates are likely not validable
```

Comment:
Do we need to add `text` (and maybe others here) to `ansiblelint.constants.FileType`?

Reply:
If we don't see an error, not really. I tested and without adding this line the tests would fail (only by adding the new example). One I added this line, it passed.

Reply:
Looks like `FileType` is already missing:
- `jinja2`
- `inventory`
- `galaxy`
- `reno` _what an unfortunate/confusing abbreviation for "release notes"_
- `vars`

Reply:
OK. That might be something to test in a future PR then.